### ISSUE TITLE
Update the existing pdf documentation or decide who will do that.

### ISSUE THREAD

**Issue by stevencarlislewalker** [stevencarlislewalker/thermopic#1]
# What needs to change:
1. The original file organization presented in section C2 is different for R packages, and so this must be updated (DONE)
2. The method of running of the software presented in section C3 now needs to be updated, given the more functional (as opposed to script-based) approach
3. The list of package dependencies is different (e.g. XLConnect is no longer required) -- full list of current dependencies in the package DESCRIPTION file (DONE)
4. It is now possible to specify user options both as the csv file format on page 5, and via standard R function arguments

**Comment by stevencarlislewalker** [comment]
I have added the source docx files to the package in inst/doc.

**Comment by nyglester** [comment]
I will make changes in the C2 and C3 - after i try running the revised system.  I have not figured that out yet, but I’m gaining speed.  I will look at other products (like Miser) to het more familiar with this environment

**Comment by stevencarlislewalker** [comment]
Great thanks @nyglester.  Note that I have already modified task 1 above (please see the current version of the word documents in the package doc directory.

You should be able to download it modify it, and upload a new version.  I’ll provide more detailed info on that once I get to a computer.

**Comment by stevencarlislewalker** [comment]
Perhaps a quick call sometime this weekend will clear up any issues you might be having with using the package, and point the way for what needs to be documented to make the package easier to pick up by new users.

**Comment by nyglester** [comment]
Yes lets chat on the weekend.  I have no fixed plans - suggest a time

N.
